source/webinterface.d

```diff
@@ -287,7 +287,17 @@ class WebInterface
 		string user = "";
 		auto location = URL(res.headers.get("Location", "https://gist.github.com/run-dlang")).path.bySegment;
 		if (!location.empty)
-			user = location.front.name;
+		{
+			auto first_loc = location.front.name;
+			// skip first slash from path
+			if(first_loc.empty) {
+				location.popFront;
```

Comment:
```suggestion
				location.popFront();
```
The whole point of `popFront` is its side effect, so better to correctly show it as a function.